tasks/common_packages.yml

```diff
@@ -34,3 +34,11 @@
       - locales
       - nfs-client
       - net-tools
+      - tree
+      - tmux
```

Comment:
Why? This barely makes sense without a good config

Reply:
Because sometimes I execute commands that take much time to complete or need multiple sessions (Yes, I could start more SSH sessions, but even a badly-configured tmux is better than that)